pkg/kubelet/cm/types.go

```diff
@@ -119,4 +123,13 @@ type PodContainerManager interface {
 
 	// IsPodCgroup returns true if the literal cgroupfs name corresponds to a pod
 	IsPodCgroup(cgroupfs string) (bool, types.UID)
+
+	// Get value of memory.usage_in_bytes for the pod Cgroup
```

Comment:
nit: Memory usage instead of .usage_in_bytes as that is v1 specific.

Reply:
ack. i'll fix it shortly with the rebase.

Reply:
fixed and reviewed. resolving